src/overlays/actors/ovl_Bg_Crace_Movebg/z_bg_crace_movebg.c

```diff
@@ -328,8 +328,8 @@ void BgCraceMovebg_ClosingDoor_Close(BgCraceMovebg* this, PlayState* play) {
     if (Math_StepToF(&this->doorHeight, 0.0f, 1.0f)) {
         if (!(this->stateFlags & BG_CRACE_MOVEBG_FLAG_PLAYER_IS_BEYOND_DOOR) &&
             !Flags_GetSwitch(play, BG_CRACE_MOVEBG_GET_SWITCH_FLAG(&this->dyna.actor) + 1)) {
-            play->unk_18845 = 1;
-            func_80169FDC(&play->state);
+            play->haltAllActors = 1;
```

Comment:
```suggestion
            play->haltAllActors = true;
```
Unless this is not a bool